### PR TITLE
Daco table row retrieval statements

### PR DESCRIPTION
**Pull request by sachin-shitole-paccar**

### CODE REVIEW

**Review by temanbrcom — APPROVED** [review]
🌗

File: server/src/test/java/org/eclipse/lsp/cobol/usecases/TestDAFTableRowAnyStatement.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2021 DAF Trucks NV.
```
Comment (grianbrcom) [inline server/src/test/java/org/eclipse/lsp/cobol/usecases/TestDAFTableRowAnyStatement.java:2]: Please, update the year.

**Review by grianbrcom — CHANGES_REQUESTED** [review]
You need just one little change for all your new files. The rest is good.

File: server/src/test/java/org/eclipse/lsp/cobol/usecases/TestDAFTableRowAnyStatement.java (hunk 1 above)
Comment (sachin-shitole-paccar) [reply server/src/test/java/org/eclipse/lsp/cobol/usecases/TestDAFTableRowAnyStatement.java:2]: good catch. will update

**Review by grianbrcom — APPROVED** [review]
:raccoon:

**Pull request merged**